builddefs/common_features.mk

```diff
@@ -335,15 +335,17 @@ ifeq ($(strip $(RGBLIGHT_ENABLE)), yes)
     endif
 
```

Comment:
Could we do it like...
```c
OPT_DEFS += -DRGBLIGHT_$(strip $(shell echo $(RGBLIGHT_DRIVER) | tr '[:lower:]' '[:upper:]'))
```
And then remove the custom block and the OPT_DEFS from the other 2?